src/main.c

```diff
@@ -66,13 +61,43 @@ int main(void)
 	GPIOB->MODER |= GPIO_MODER_MODER3_0; // make bit3  an output
 	GPIOB->MODER &= ~GPIO_MODER_MODER3_1; // make bit3  an output
 
+  adc_status = adc_up();
+  if (ADC_ERROR(adc_status)) {
+    assert_failed(__FILE__, __LINE__);
+  }
+
+  __enable_irq();
+
 	while (1) {
 		GPIOB->ODR |= GPIO_MODER_MODER1_1;
 		delay(500000);
+    /* TODO Fix adc_conversion, currently this returns a timeout error */
+    adc_status = adc_convert(ADC_CONVERT_PA1);

```

Comment:
I'm not sure if this is the right pin or where this is physically on the breakout board.